plugins/tileacq.py

```diff
@@ -568,6 +573,45 @@ def _check_fov(self, das, sfov):
             sfov = asfov
         return sfov
 
+    def _memory_check(self, _):
```

Comment:
Write it as `_memory_check(self, _=None)`, so that it's possible to simply call it with `._memory_check()`